rst_files/mccall_model.rst

```diff
@@ -602,30 +515,23 @@ Repeat a large number of times and take the average
 
 Plot mean unemployment duration as a function of :math:`c` in ``c_vals``
 
-
 Solutions
 ==========
 
-
 Exercise 1
 ----------
 
 Here's one solution
 
-
-
 .. code:: julia
 
     function compute_stopping_time(w_bar; seed=1234)
-
         Random.seed!(seed)
         stopping_time = 0
-
         t = 1
         while true
```

Comment:
Is there a hidden error here if they change the parameters and have parameters such that w_bar is never binding?  Might want to make it more robust.

Reply:
You're right. I'll include a check that the `w_bar` is inside the distribution.